playback/src/lib.rs

```diff
@@ -16,3 +16,5 @@ pub mod player;
 pub const SAMPLE_RATE: u32 = 44100;
 pub const NUM_CHANNELS: u8 = 2;
 pub const SAMPLES_PER_SECOND: u32 = SAMPLE_RATE as u32 * NUM_CHANNELS as u32;
+pub const SAMPLES_PER_MS: f64 = SAMPLE_RATE as f64 / 1000.0;
+pub const MS_PER_SAMPLE: f64 = 1000.0 / SAMPLE_RATE as f64;
```

Comment:
These are not consistent with `SAMPLES_PER_SECOND` above, that looks at an entire frame with two channels.

Reply:
We can do `PAGES_PER_MS` and `MS_PER_PAGE` for vorbis at least seeks are done in pages from what I understand. Basically a page is the same as a frame in ALSA. Channel count does not matter.

Reply:
No but the constant above was introduced for the dynamic limiter, which does keep track of channels (or actually, interleaved samples).

Reply:
> No but the constant above was introduced for the dynamic limiter, which does keep track of channels (or actually, interleaved samples).

I guess I don't understand what you mean or what a new constant has to do with existing constants? SAMPLES_PER_SECOND has nothing to do with either of the 2 new constant. SAMPLE_RATE does.

Reply:
Done.